CMakeLists.txt

```diff
@@ -197,8 +200,7 @@ set(CONTROL_SOURCES
         ${PROJECT_SOURCE_DIR}/src/control/ballHandling/DribbleBackwards.cpp
         ${PROJECT_SOURCE_DIR}/src/control/ballHandling/DribbleForwards.cpp
         ${PROJECT_SOURCE_DIR}/src/control/ballHandling/RotateAroundBall.cpp
-        ${PROJECT_SOURCE_DIR}/src/control/ballHandling/RotateWithBall.cpp
-        )
+        ${PROJECT_SOURCE_DIR}/src/control/ballHandling/RotateWithBall.cpp)
```

Comment:
Not huge, but please put keep the ) on the next line. This prevents merge conflicts when multiple people are working on different branches.